src/DataSets/DataSets.component.js

```diff
@@ -173,10 +181,21 @@ const DataSets = React.createClass({
     openLogs() {
         // Retrieve the logs and save them in this.state.logs, and set
         // this.state.logsObject to a description of their contents.
+        const title = this.tr("logs") + " (" + this.tr("all") + ")";
+        this.setState({
+            logsObject: title,
+            logs: null,
+        });
         getLogs().then(logs => {
-            this.setState({logs: _(logs).orderBy('date', 'desc').value().map(LogEntry)});
+            this.setState({logs: _(logs).orderBy('date', 'desc').value()});
```

Comment:
Does it makes a local variable for the repeated expression?

Reply:
I'm not sure I follow your comment.

If you mean the `const title = ...`, I added to emphasize that the object just contains the title.

If you mean why I removed the `.map(LogEntry)` part, it's so we store the logs themselves and can concatenate (and filter and sort) more when more pages are requested.

Reply:
Dismiss my comment, I was tired, I thought they were two different lines!